src/app/cases/cases.module.ts

```diff
@@ -49,6 +50,8 @@ import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
     CaseEditPageComponent,
     CaseEditFormComponent,
     CaseEditSubmitComponent,
+    CaseCreateComponent,
+    CaseProgressComponent,
```

Comment:
Why are those 2 declarations required given that they are not used in this app?

Reply:
Looks like accidental check in - will clean up.